neoscore/western/chordrest.py

```diff
@@ -333,6 +333,38 @@ def extra_attachment_point(self) -> Point:
         x = notehead.x + bounding_rect.x + (bounding_rect.width / 2)
         return Point(x, y)
 
+    def mid_stem_attachment_point(self) -> Point:
```

Comment:
separate test coverage needed for this too. some test cases:
- rest
- stemless note durations
- single-note stemmed CRs up and down
- multi-note stemmed CRs (probably up and down too for good measure)

Reply:
all done :)